src/printer.rs

```diff
@@ -423,14 +428,16 @@ impl Printer {
 
     pub fn print_response_body(
         &mut self,
-        mut response: Response,
+        response: &mut Response,
```

Comment:
Incidentally, this is just the change I needed in #240 i.e not consuming Response so `response_meta` can be accessed later.